views/check-your-answers.html

```diff
@@ -18,7 +18,7 @@ <h1 class="govuk-heading-xl">Check your answers before sending your application<
       <br>
       {% include "includes/check-your-answers/identity-checks.html" %}
       <br>
-      {% include "includes/check-your-answers/entity.html" %}
+      {% include "./update/includes/entity.html" %}
```

Comment:
We should move /udate/include/entity.html to includes/entity.html and then call it from both the check-your-answers page and our new update entity review page. In future we can use it for the update check-your-answers too.